lib/Ride.dart

```diff
@@ -182,37 +164,38 @@ class _RideCardState extends State<RideCard> {
                       Container(
                         decoration: BoxDecoration(
                           shape: BoxShape.circle,
-                          boxShadow: [dropShadow],
+                          boxShadow: [CarriageTheme.shadow],
                         ),
                         child: CircleAvatar(
                           radius: 24,
                           //TODO: replace with rider's image
                           backgroundImage:
-                              AssetImage('assets/images/terry.jpg'),
+                          AssetImage('assets/images/terry.jpg'),
                         ),
                       ),
                       SizedBox(width: 16),
                       Column(
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: [
                             Text(widget.ride.rider.firstName,
-                                style: TextStyle(
-                                    fontSize: 20, fontWeight: FontWeight.bold)),
+                                style: CarriageTheme.title3
```

Comment:
Nit: is there a way to make the theme titles more descriptive? Like when would title3 be used? if that's not possible due to changing designs feel free to ignore this

Reply:
Also if title3 or gray3 are the names from the design system, definitely don't change it then.

Reply:
Talked about this on Slack but just for the record if anyone else has the same question, the theme titles all correspond to the titles in the design system in Figma where they're called title 1/2/3 and gray 1/2/3!